modules/apps/27-interchain-accounts/host/client/cli/tx.go

```diff
@@ -21,13 +22,13 @@ const (
 
 func generatePacketDataCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "generate-packet-data [message]",
-		Short: "Generates protobuf encoded ICA packet data.",
-		Long: `generate-packet-data accepts a message string and serializes it using protobuf
-into packet data which is outputted to stdout. It can be used in conjunction with send-tx"
-which submits pre-built packet data containing messages to be executed on the host chain.
-`,
-		Example: fmt.Sprintf(`%s tx interchain-accounts host generate-packet-data '{
+		Use:   "generate-packet-data [encoding] [message]",
```

Comment:
Should encoding be added as an optional flag for backwards compatibility? Like `--proto3json` or `--protobuf`